internal/cli/test.go

```diff
@@ -269,7 +269,6 @@ func testTokenCmd(cli *cli) *cobra.Command {
 	cmd.SetUsageTemplate(resourceUsageTemplate())
 	cmd.Flags().BoolVar(&cli.force, "force", false, "Skip confirmation.")
 	cmd.Flags().BoolVar(&cli.json, "json", false, "Output in json format.")
-	testAudience.IsRequired = true
```

Comment:
Requiring the audience precluded the ability to test authentication in non-interactive mode.

Reply:
It looks like the initial change of toggling this to be required is in #221 otherwise it generated invalid tokens. Testing locally it still seems to be that dropping audience will generate an invalid token due to a missing payload.

Reply:
Let's talk about this later.

Reply:
**Update:** Turns out that this marks the audience flag required for both the `test token` and `test login` commands but it should only be requiring for `test token` command. Will be fixing in a following commit.